src/main/java/com/derongan/minecraft/deeperworld/DeeperCommandExecutor.kt

```diff
@@ -1,62 +1,70 @@
 package com.derongan.minecraft.deeperworld
 
+import com.derongan.minecraft.deeperworld.MinecraftConstants.FULL_DAY_TIME
 import com.derongan.minecraft.deeperworld.config.DeeperConfig
 import com.derongan.minecraft.deeperworld.services.WorldManager
 import com.derongan.minecraft.deeperworld.services.canMoveSections
 import com.mineinabyss.idofront.commands.CommandHolder
+import com.mineinabyss.idofront.commands.arguments.booleanArg
 import com.mineinabyss.idofront.commands.arguments.intArg
 import com.mineinabyss.idofront.commands.execution.ExperimentalCommandDSL
 import com.mineinabyss.idofront.commands.execution.IdofrontCommandExecutor
+import com.mineinabyss.idofront.commands.execution.stopCommand
 import com.mineinabyss.idofront.commands.extensions.actions.playerAction
 import com.mineinabyss.idofront.messaging.info
 import com.mineinabyss.idofront.messaging.success
 
 @ExperimentalCommandDSL
 object DeeperCommandExecutor : IdofrontCommandExecutor() {
     override val commands: CommandHolder = commands(deeperWorld) {
-        "sectionoff" {
-            playerAction {
-                player.canMoveSections = false
-                sender.success("Automatic TP disabled for ${player.name}")
-            }
-        }
-        "sectionon" {
-            playerAction {
-                player.canMoveSections = true
-                sender.success("Automatic TP enabled for ${player.name}")
+        ("deeperworld" / "dw") {
+            "tp"(desc = "Enables or disables automatic teleports between sections for a player") {
+                val state by booleanArg()
+                playerAction {
+                    if (state) {
+                        player.canMoveSections = true
```

Comment:
Use `player.canMoveSections = state`. I'd also maybe do `val msg = if(state) "enabled" else "disabled"` and have one line sending that.